core/domain/question_services.py

```diff
@@ -20,6 +20,7 @@
 
 from core.domain import question_domain
 from core.platform import models
+from core.domain import user_services
 import feconf
 
 (question_models,) = models.Registry.import_models([models.NAMES.question])
```

Comment:
Since you are declaring CMD_CREATE_NEW in question_domain, can you remove it from here?

Reply:
This is till there, right?

Reply:
I've removed this declaration and used the previous one.